### PR TITLE
try to insert wihout create database before (MONGO sinks)

### PR DESCRIPTION
**Pull request by AlvaroVega**
possible fix for https://github.com/telefonicaid/fiware-cygnus/issues/2156

### CODE REVIEW

**Comment by AlvaroVega** [comment]
This is an improvement respect the original implementation:

https://github.com/telefonicaid/fiware-cygnus/blob/24e235bf1511e413145e7bc2f96ab213cbc0eb92/cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIMongoSink.java#L246-L249

This is not doing listCollection before create collection, but createCollection allow apply configuration for collection (i.e. related with expiration date, and so on) even if collection exists

File: cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIMongoSink.java [hunk 1]
```diff
@@ -283,25 +283,11 @@ private void persistAggregation(NGSIGenericAggregator aggregator) throws CygnusP
         String collectionName = aggregator.getCollectionName(enableLowercase);
         LOGGER.info("[" + this.getName() + "] Persisting data at NGSIMongoSink. Database: "
                 + dbName + ", Collection: " + collectionName + ", Data: " + aggregation.toString());
-        
         try {
-            // try insert without create database and collection before
+            backend.createCollection(dbName, collectionName, collectionsSize, maxDocuments, dataExpiration);
```
Comment (fgalan) [inline cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIMongoSink.java:287]:
```suggestion
            // createCollection is an idempotent operation so we can safely run it each time a new doc is going to be inserted
            backend.createCollection(dbName, collectionName, collectionsSize, maxDocuments, dataExpiration);
```

File: cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSISTHSink.java [hunk 2]
```diff
@@ -372,32 +372,14 @@ public void persist(String sinkName) throws CygnusBadConfiguration, CygnusPersis
                         + "," + numericAggr.getNumSamples() + "]");
 
                 try {
-                    // try insert without create database and collection before
+                    // try insert without create database before
```
Comment (fgalan) [inline cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSISTHSink.java:375]:
```suggestion
                    // createCollection is an idempotent operation so we can safely run it each time a new doc is going to be inserted        
```

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**